src/tests/homework.spec.js

```diff
@@ -0,0 +1,136 @@
+import { expect, test } from "@playwright/test";
+
+async function openRegistrationPage(page) {
+  await page.goto("/registrace");
+}
+
+function getRegistrationPageHeader(page) {
+  return page.getByRole("heading", { level: 1 });
+}
+
+function getInputName(page) {
+  return page.locator("#name");
+}
+
+function getInputEmail(page) {
+  return page.locator("#email");
+}
+
+function getInputPassword(page) {
+  return page.locator("#password");
+}
+
+function getInputPasswordConfirmation(page) {
+  return page.locator("#password-confirm");
+}
+
+function getSubmitButton(page) {
+  return page.locator("button[type='submit']");
+}
+
+async function expectRegistrationForm(page) {
+  await expect(getInputName(page)).toBeVisible();
+  await expect(getInputEmail(page)).toBeVisible();
+  await expect(getInputPassword(page)).toBeVisible();
+  await expect(getInputPasswordConfirmation(page)).toBeVisible();
+  await expect(getSubmitButton(page)).toBeVisible();
+}
+
+async function registration(page, name, username, password, confirmpassword) {
```

Comment:
Návrh na více vypovídající název
```suggestion
async function registerUser(page, name, username, password, confirmpassword) {
```